core/src/br/cefetmg/games/AnimatedSprite.java

```diff
@@ -0,0 +1,88 @@
+package br.cefetmg.games;
+

```

Comment:
Temos uma classe AnimatedSprite já, do pacote gdx-utils. Temos uma MultiAnimatedSprite também. Qual é a finalidade desta? Avaliem se ela está sendo utilizada por mais de 1 minigame. Se for o caso e as AnimatedSprite/MultiAnimatedSprite que já existem não atenderem a mesma demanda desta, coloquem-na como uma classe interna do MiniGame que a utiliza (se for apenas 1) ou então dentro do pacote `.graphics` (onde a MultiAnimatedSprite está. Além disso, mude o nome dela para que não seja o mesmo da AnimatedSprite da biblioteca gdx-utils.